docs/wave.rst

```diff
@@ -42,7 +43,7 @@ receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`da
   data.set_receiver_pos(receiver_coords)
   data.set_shape(nt, 101)
 
```

Comment:
origin = (0, 0, 0) in 3D that is the position in meters of the postion of index (0, 0 ,0)
spacing, size of the discrete grid, distance in meters between two consecutive grid point in each direction
dimension : size of the model in number of grid points.